docs/Over_OSPO-NL.md

```diff
@@ -1,7 +1,29 @@
-```{warning}
-De documentatie is nog volop in ontwikkeling
-```
-
 # Over OSPO-NL
  
-TO DO
\ No newline at end of file
+OSPO staat voor Open Source Program Office.
+De toevoeging NL doelt op de Nederlandse samenwerking om kennis en best practices te delen
+voor het opzetten en uitvoeren van OSPO's (of OSPOs) bij organisaties in Nederland.
+Het is niet van belang of deze organisatie commerciële bedrijven zijn of overheidsinstanties.
+
+Het initiatief van deze community en samenwerking is ontstaan naar aanleiding van de
+[OSPOlogy.live](https://community.linuxfoundation.org/events/details/lfhq-ospology-european-chapter-presents-ospologylive-share-learn-netherlands/)
+van januari 2023.
+Dit event werd georganiseerd door de [Linux Foundation](https://www.linuxfoundation.org/) en gehost door [Alliander](https://www.alliander.com/en/open-source/projects/).
+
+OSPO-NL is een open community welke gestart is door [Alliander](https://www.alliander.com/en/open-source/projects/),
+[Belastingdienst](https://github.com/belastingdienst) en [Kadaster](https://github.com/kadaster).

```

Comment:
Ik heb hier maar de GitHub Orgs van BLD en Kdstr gebruikt ... omdat wij (nog?) geen mooie open source pagina's hebben op onze company websites. Ik wil bij ons (Kdstr) werk maken van een [GitHub Org Profile](https://docs.github.com/en/organizations/collaborating-with-groups-in-organizations/customizing-your-organizations-profile#adding-a-public-organization-profile-readme) om daar toch enige invulling aan te gaan geven (op relatief korte termijn). Hoe kijk jij daar tegenaan, @RietveldBLD ?

Reply:
@marcvanandel er wordt samengewerkt met Communicatie aan een uitwerking zodat er ook een publieke GitHub profiel zichtbaar wordt. De profielpagina van BZK is ook nog in ontwikkeling

Reply:
Check. Maar voor nu kan ik wel alvast deze URL gebruiken?